src/tasks/peripherals/task_nfc.c

```diff
@@ -143,4 +144,4 @@ ruuvi_driver_status_t task_nfc_init (void)
 {
     return  RUUVI_DRIVER_SUCCESS;
 }
-#endif
\ No newline at end of file
+#endif
```

Comment:
While doing changes, please add a newline to the end of file. It fixes one automatic check complaint.